src/org/usfirst/frc/team2473/robot/commands/PointTurn.java

```diff
@@ -16,91 +16,73 @@
 
 public class PointTurn extends Command {
 	
-	private static double SLOW_POWER = 0.06;
-	
 	private double leftPower;
 	private double rightPower;
-	private double leftPowerSlow;
-	private double rightPowerSlow;
 	
 	private boolean isClockwise;
-	private double prevDegrees;
-	private double degreesGoal;
-	private double initial;
+	private double prevAngle;
+	private double angleGoal;
+	private double initialAngle;
 	
 	
 	public PointTurn(double degrees, double power) {
 		requires(Robot.driveSubsystem);
 		
-		if (power < 0) {
-			throw new IllegalArgumentException("Power must be a positive scalar for point turn!");
-		}
+		if (power < 0) throw new IllegalArgumentException("Power must be a positive scalar for point turn!");
 		
 		isClockwise = degrees > 0;
 		
 		this.leftPower = isClockwise ? power : -power;
 		this.rightPower = -leftPower;
 		
-		this.leftPowerSlow = isClockwise ? SLOW_POWER : -SLOW_POWER;
-		this.rightPowerSlow = -leftPowerSlow;
-		
-		prevDegrees = Devices.getInstance().getNavXGyro().getAngle();
-		this.initial = prevDegrees;
-
-		
-		this.degreesGoal = prevDegrees + degrees;
-
 		
+		prevAngle = Devices.getInstance().getNavXGyro().getAngle();
+		this.initialAngle = prevAngle;
+		this.angleGoal = prevAngle + degrees;
 	}
 	
 	public void setPower(double power) {
-		if (power < 0) {
-			throw new IllegalArgumentException("Power must be a positive scalar for point turn!");
-		}
+		if (power < 0) throw new IllegalArgumentException("Power must be a positive scalar for point turn!");
 		this.leftPower = isClockwise ? power : -power;
-		this.rightPower = -leftPower;
-		
+		this.rightPower = -leftPower;	
 	}
-
+	
```

Comment:
make a javadoc comment here to let the user know that is will always be positive